docs/tutorial_trajectories.md

```diff
@@ -144,39 +160,37 @@ You can use the same environment as before.
 You can use this script to load the trajectory data into your database.
 
 Instructions on how to run this script can be found in the accompanying [README.md](https://github.com/JPBergsma/Export_traj_to_mongo/blob/master/README.md) file.
-If you have not restarted since installing MongoDB you still need to start MongoDB with: `sudo systemctl start mongod`
-You can check whether mongo db is running you can use: `sudo systemctl status mongod`  
+If you have not restarted since installing MongoDB you still need to start MongoDB with: `systemctl start mongod`
+You can check whether MongoDB is running you can use: `systemctl status mongod`  
 
 ### Validation
 
-To test the setup, you can go to optimade-python-tools folder and run:
-`uvicorn optimade.server.main:app --reload --port=5000`
-This will start the optimade server.
+To launch the OPTIMADE API server and test the setup, you can go to optimade-python-tools folder and run:
+
+```uvicorn optimade.server.main:app --reload --port=5000```
+
+By adding the --reload flag, the server is automatically restarted when code is changed as you develop your server.
 Next, you can run `optimade-validator http://localhost:5000` to validate the setup of your database.
-At the moment, the validator may still give a "'StrictFieldInfo' object is not subscriptable" error. You can ignore this error for now.
-Errors under INTERNAL FAILURES indicate problems with the validator itself and not with the server setup. You can report those [here](https://github.com/JPBergsma/optimade-python-tools/issues).
-More details about validating your server can be found in [validation.md](https://github.com/JPBergsma/optimade-python-tools/blob/optimade_python_tools_trajectory_0.1/docs/concepts/validation.md).
 
-### Deployment
 
-If you are satisfied with how things are , you can run the server with:
-`uvicorn optimade.server.main:app --reload --port=5000 --host=0.0.0.0` for it to be reachable from any IPv4 adress and
-`uvicorn optimade.server.main:app --reload --port=5000 --host='::'` for IPv6
```

Comment:
I think discussing `--host` here is a bit confusing and "reachable from any IPv4" address is misleading I think as it is only an internal networking distinction. You also don't mention that the database must be visible from this server too. 

I think I would suggest adding a link to the container deployment docs at https://www.optimade.org/optimade-python-tools/latest/deployment/container/ as they should be generally useful.

Reply:
The trajectory branch uses python packages that are not in the standard optimade container. (h5py and NumPy) 
So I think I should create a separate container image that already includes these python packages, or at least give instructions on how to install them in the docker container, before I reference the instructions for the docker container.

Reply:
I have added an extra section to explain how you can run the trajectory version of the optimade-python-tools in a docker container. I ran into some problems when I tried to use the local code in a docker container(see issue #1589), so instead I let them create their own container.